src/main/java/build/buildfarm/worker/memory/OperationQueueClient.java

```diff
@@ -55,7 +55,7 @@ void match(MatchListener listener) throws InterruptedException {
 
           @Override
           public boolean getMatched() {
-            return matched;
+            return !matched;
```

Comment:
dont think these are a good idea.  we would need to change the function name to is getUnmatched so it makes sense,  
but someone could along and then use `if (!x.getUnmatched())`.  So I'm not really sure what the analyzer is trying to achieve.